common/lib/xmodule/xmodule/js/src/video/08_video_speed_control.js

```diff
@@ -109,6 +109,7 @@ function (Iterator) {
             speedsContainer.html(speedsList.join(''));
             this.speedLinks = new Iterator(speedsContainer.find('.speed-option'));
             this.state.el.find('.secondary-controls').prepend(this.el);
+            this.setActiveSpeed("1.0"); // default state on initial render
```

Comment:
Got it. I will take another look at this. Maybe I will check for the existence first and use that value if it's present, with a default backup to 1.0x. This may also be set as a cookie.